### PR TITLE
Fix for bos send with scid alias

### PR DESCRIPTION
**Pull request by niteshbalusu11**
Signed-off-by: Nitesh Balusu <niteshbalusu@icloud.com>

Get peers is returning strings for feature bit numbers, i added a type conversion for now. If getPeers is fixed this part can be removed.

```
            .map(n => {
              return {
                bit: Number(n.bit),
                is_known: n.is_known,
                is_required: n.is_required,
                type: n.type,
              }
            })
```

### CODE REVIEW

**Comment by alexbosworth** [comment]
Hmm ok so the lightning library should be fixed then?

File: network/probe_destination.js [hunk 1]
```diff
@@ -356,7 +395,7 @@ module.exports = (args, cbk) => {
         'outgoingChannelId',
         'to',
         ({getFeatures, getIcons, messages, outgoingChannelId, to}, cbk) =>
-      {
+      {
```
Comment (alexbosworth) [inline network/probe_destination.js:398]:
```suggestion
      {
```

**Comment by alexbosworth** [comment]
I changed ln-service to return numbers

**Comment by niteshbalusu11** [comment]
Ok I'll update bos and remove type conversation.

**Comment by niteshbalusu11** [comment]
Updated ln-service and removed type conversion.

**Pull request merged**